static/src/js/components/GranuleResults/GranuleResultsDataLinksButton.scss

```diff
@@ -9,7 +9,7 @@
     cursor: pointer;
 
     .button__icon {
-      color: rgba($color__spacesuit-white, 0.7);
+      color: $color__carbon--40;
```

Comment:
this should maybe be `$color__carbon--50;` since that is what we are using in `static/src/js/components/CopyableText/CopyableText.scss`

Reply:
I made it the same color as the nearby copy button so they all match this button I mean:
![image](https://github.com/user-attachments/assets/5e0dd792-327d-4e43-8c87-028e42d35c1a)